server/index.ts

```diff
@@ -86,7 +94,12 @@ app.use(
 		crossOriginEmbedderPolicy: false,
 		contentSecurityPolicy: {
 			directives: {
-				'connect-src': MODE === 'development' ? ['ws:', "'self'"] : null,
+				'connect-src':
+					MODE === 'development'
+						? ['ws:', "'self'"]
+						: process.env.SENTRY_DSN
+						? ['*.ingest.sentry.io']
```

Comment:
If self-hosted, would the URL here not be different? Some blockers (https://github.com/easylist/easylist/issues/6963) will block sentry.io, and therefor self-hosted/tunnel can be used

Reply:
@TryingToImprove Yes, the self-hosted or tunnel URL would be different. I can add a comment.